CookSmart/CookSmart.xcodeproj/project.pbxproj

```diff
@@ -876,7 +876,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 10;
-				DEVELOPMENT_TEAM = T36GJ569RT;
+				DEVELOPMENT_TEAM = 5B7P72SMLP;
```

Comment:
what do people usually do about this :(

Reply:
We usually don't change these. The development team should be Olga's Apple Developer team now